packages/extension-browser/src/devtools/views/pages/analyze.css

```
@@ -41,6 +41,12 @@
     font-size: var(--font-size-sm);
 }
 
+.message-title {
```

I've been following the [CSS Modules recommendation of camelCased names](https://github.com/css-modules/css-modules#naming) to allow direct object access in our script files.
```suggestion
.messageTitle {
```